LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Google Inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Is removing the year intentional ?

Reply:
yes, the license needs to be kept exactly as is with Copyright [yyyy] [name of copyright owner]